apps/cms/models.py

```diff
@@ -1,22 +1,68 @@
 from django.db import models
 from modelcluster.fields import ParentalKey
 from modelcluster.models import ClusterableModel
-from wagtail.wagtailadmin.edit_handlers import FieldPanel
-from wagtail.wagtailadmin.edit_handlers import InlinePanel
-from wagtail.wagtailadmin.edit_handlers import PageChooserPanel
-from wagtail.wagtailcore.fields import RichTextField
+from wagtail.wagtailadmin import edit_handlers
+from wagtail.wagtailcore import blocks
+from wagtail.wagtailcore import fields
 from wagtail.wagtailcore.models import Orderable
 from wagtail.wagtailcore.models import Page
+from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
 from wagtail.wagtailsnippets.models import register_snippet
 
+from adhocracy4.projects.models import Project
+
+
+class CallToActionBlock(blocks.StructBlock):
+    body = blocks.RichTextBlock()
+    link = blocks.CharBlock()
+    link_text = blocks.CharBlock(max_length=50, label='Link Text')
+
+    class Meta:
+        template = 'meinberlin_cms/blocks/cta_block.html'
+
+
+class ColumnsBlock(blocks.StructBlock):
+    columns_count = blocks.ChoiceBlock(choices=[
+        (2, 'Two columns'),
+        (3, 'Three columns'),
+        (4, 'Four columns'),
+    ], default=2)
+
+    columns = blocks.ListBlock(
+        blocks.RichTextBlock(label='Column body'),
+    )
+
+    class Meta:
+        template = 'meinberlin_cms/blocks/columns_block.html'
+
 
 class HomePage(Page):
-    body = RichTextField(blank=True)
+    body = fields.StreamField([
+        ('paragraph', blocks.RichTextBlock()),
```

Comment:
I agree that it is a nice addition, but I doubt that it would work well in production without the `.l-center-6`. You can also just leave it and we get back to it if needed.